packages/clay-color-picker/webpack.config.js

```diff
@@ -0,0 +1,20 @@
+const path = require('path');
```

Comment:
For the demo to work here without having to give `yarn start`, we will have to make some modifications here. Here's how I did it on [ClayDatePicker](https://github.com/liferay/clay/blob/9630c7157d9cb0b8c270a26353ceb630f0045c08/packages/clay-date-picker/webpack.config.js).